src/pages/Network.tsx

```diff
@@ -30,29 +30,37 @@ const NetworkView: React.StatelessComponent<Props> = ({ members }) => {
   members.forEach(member => {
     graphNodes.push(
       <ForceGraphNode
-        key={member.uid}
-        node={{ id: member.uid, label: member.username }}
+        key={member.get("memberId")}
+        node={{ id: member.get("memberId"), label: member.get("username") }}
         fill="gray"
       />
     );
-    Object.keys(member.trustsSet).forEach(trustedUid => {
+    for (const trustedUid of member.get("trusts").toArray()) {
```

Comment:
Pretty sure `Immutable.List` has `forEach` and `map` functions, don't need to make it array.

```javascript
member.get("trusts").forEach(trustedUid => { ... })
```